bower.json

```diff
@@ -1,16 +1,43 @@
 {
   "name": "paper-dropdown-menu",
-  "private": false,
+  "version": "1.0.0",
+  "description": "",
```

Comment:
Needs a description or else the catalog is a sad panda (I think I missed this for `iron-dropdown` as well)